packages/preview/tutor/0.3.0/README.md

```diff
@@ -0,0 +1,122 @@
+# tutor
+
+Utilities to write exams and exercises with integrated solutions. Set the variable `sol=true` to display the solutions of a document.
+
+Currently the following features are supported:
+
+* Automatic total point calculation through the `#points()` and `#totalpoints()` functions.
+* Checkboxes that are either blank or show the solution state using eg. `#checkbox(cfg, true)`.
+* Display blank lines allowing students to write their answer using eg. `#lines(cfg, 3)`.
+* A proposition for a project structure allowing self-contained exercises and a mechanism to show or hide the solutions of an exercise.
+
+## Usage
+
+### Minimal Example
+
+```typst
+#import "@local/tutor:0.2.0": points, totalpoints, lines, checkbox
+
+#let cfg = default_config()
```

Comment:
The idiomatic case would be `default-config` in case you want to change that. (It's up to you.)

Reply:
Still have to get used to kebab-case, but thanks for the hint. Please find the updates below.